docs/source/tutorials/psij.ipynb

```diff
@@ -0,0 +1,547 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.wait()\n",
+    "print(job.status.state)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configuring a Job\n",
+    "The previous examples specify `executable='/bin/date'` tells PSI/J that we want to run the `/bin/date` command. The superclass is `JobSpec`, which allows you to configure the following parameters used in the instance of `Job`:\n",
+    "- arguments for the job executable\n",
+    "- environment the job is runnning in\n",
+    "- destination for standard output and error streams\n",
+    "- resource requirements for the job's execution\n",
+    "- accounting details to be used\n",
+    "\n",
+    "Relevant Docs: [JobSpec](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_spec.JobSpec)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Job Arguments\n",
+    "The executable’s command line arguments to be used for a job are specified as a list of strings in the arguments attribute of the `JobSpec` class. For example, our previous `/bin/date` job could be changed to request UTC time formatting:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "environment = 'local'\n",
+    "ex = JobExecutor.get_instance(environment)\n",
+    "job = Job(JobSpec(executable='/bin/date', arguments=['-utc', '--debug']))\n",
+    "\n",
+    "print(str(job.status))\n",
+    "ex.submit(job)\n",
+    "job.wait()\n",
+    "print(str(job.status))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: `JobSpec` can also be added incrementally"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from psij import JobSpec\n",
+    "\n",
+    "spec = JobSpec()\n",
+    "spec.executable = '/bin/date'\n",
+    "spec.arguments = ['-u']\n",
+    "job = Job(spec)\n",
+    "\n",
+    "print(str(job.status))\n",
+    "ex.submit(job)\n",
+    "job.wait()\n",
+    "print(str(job.status))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Job Environment\n",
+    "The job environment sets the environment variables for a job before it is launched. This is the equivalent of exporting `FOO=bar` on the command line before running a command. These environment variables are specified as a dictionary of string key/value pairs:\n",
+    "\n",
+    "*Note: Environment variables specified this way will overwrite settings from your shell initialization files (e.g., ~/.bashrc), including from any modules loaded in the default shell environment.*"
```

Comment:
I'm not entirely sure what this means. It would seem obvious that if I set `FOO=bar` before running a job then `FOO` would be `bar` and not something else.

Reply:
This comment has been addressed

Reply:
Sorry for not being clear. My problem was with the sentence "Environment variables specified this way will overwrite settings from your shell initialization files (e.g., ~/.bashrc), including from any modules loaded in the default shell environment.*". It would seem useless to set an environment variable in the specification only to have it completely ignored and the bashrc versions be used. So yes, it will override environment variables set before with the same name and I don't think it need clarification.